wallets/index.js

```diff
@@ -49,53 +66,40 @@ export function useWallet (name) {
     }
   }, [me, wallet, config, logger, status])
 
-  const enable = useCallback(() => {
-    enableWallet(name, me)
-    logger.ok('wallet enabled')
-  }, [name, me, logger])
-
-  const disable = useCallback(() => {
-    disableWallet(name, me)
-    logger.info('wallet disabled')
-  }, [name, me, logger])
-
   const setPriority = useCallback(async (priority) => {
     if (_isConfigured && priority !== config.priority) {
       try {
-        await saveConfig({ ...config, priority })
+        await saveConfig({ ...config, priority }, { logger })
       } catch (err) {
         toaster.danger(`failed to change priority of ${wallet.name} wallet: ${err.message}`)
       }
     }
-  }, [wallet, config, logger, toaster])
+  }, [wallet, config, toaster])
 
   const save = useCallback(async (newConfig) => {
+    // testConnectClient should log custom INFO and OK message
+    // testConnectClient is optional since validation might happen during save on server
+    // TODO: add timeout
+    let validConfig
     try {
-      // testConnectClient should log custom INFO and OK message
-      // testConnectClient is optional since validation might happen during save on server
-      // TODO: add timeout
-      const validConfig = await wallet.testConnectClient?.(newConfig, { me, logger })
-      await saveConfig(validConfig ?? newConfig)
-      logger.ok(_isConfigured ? 'wallet updated' : 'wallet attached')
+      validConfig = await wallet.testConnectClient?.(newConfig, { me, logger })
     } catch (err) {
-      const message = err.message || err.toString?.()
-      logger.error('failed to attach: ' + message)
+      logger.error(err.message)
       throw err
     }
-  }, [_isConfigured, saveConfig, me, logger])
+    await saveConfig(validConfig ?? newConfig, { logger })
```

Comment:
Only `testConnectClient` is now wrapped in `try/catch` because only its errors need to be logged.  `saveConfig` already handles logging within.